### PR TITLE
awsSts: add awsSts

### LINKED ISSUE #502: [Request] Add awsSts
https://github.com/NearlyUnique/awsSts

A tool to login with temporary login credentials to AWS, for when SAML is used. It's provided as a single executable to run in their Releases page.

### PR DESCRIPTION
**Pull request by Bassetts**
Resolves #502 😄

### CODE REVIEW

File: bucket/awsSts.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "license": "MIT",
+    "url": "https://github.com/NearlyUnique/awsSts/releases/download/0.8/awsSts-0.8-windows.exe#/awsSts.exe",
+    "hash": "c5691ef95bd870b61ca451d176c7506cdbaf59ef9e9c534ae0c510016c7bc40a",
+    "bin": [
+        [
+            "awsSts.exe",
+            "awsSts2"
+        ]
+    ],
+    "checkver": {
```
Comment (linsui) [inline bucket/awsSts.json]:
```suggestion
    "checkver": "github",
```

File: bucket/awsSts.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "homepage": "https://github.com/NearlyUnique/awsSts",
+    "description": "Create temporary login session for AWS CLI",
+    "version": "0.8",
+    "license": "MIT",
+    "url": "https://github.com/NearlyUnique/awsSts/releases/download/0.8/awsSts-0.8-windows.exe#/awsSts.exe",
+    "hash": "c5691ef95bd870b61ca451d176c7506cdbaf59ef9e9c534ae0c510016c7bc40a",
+    "bin": [
+        [
+            "awsSts.exe",
+            "awsSts2"
```
Comment (linsui) [inline bucket/awsSts.json]: Why rename it?

Comment (Bassetts) [reply bucket/awsSts.json]: The usage message for the app calls the command `awsSts2` on newer versions. I originally just had the file as `awsSts2.exe` but thought it was nicer to have the alias so both `awsSts` and `awsSts2` were available.

image

Comment (linsui) [reply bucket/awsSts.json]: Then you can add both of them.

```
   "bin": [
        "awsSts.exe",
        [
            "awsSts.exe",
            "awsSts2"
        ]
    ],
```

Comment (Ash258) [reply bucket/awsSts.json]: If you want both, then it have to look like this:

```suggestion
    "bin": [
        "awsSts.exe",
        [
            "awsSts.exe",
            "awsSts2"
        ]
    ],
```

**Pull request merged**